src/app.py

```diff
@@ -5,41 +5,82 @@
 import os
 from util.serialize_image import deserialize_image
 from flask import Flask, request, send_file
+from imagebox import ImageBox
 
 import ai_mediapipe
 from prediction import Prediction
 
+RPS_SCHEMA_HOSTNAME = os.environ.get("RPS_SCHEMA_HOSTNAME", "http://localhost:5000/")
+
 app = Flask(__name__)
 ai_mediapipe.load_model("gesture_train.csv")
 
-def weighted_prediction(prediction_list: list) -> Prediction:
-    """Takes the list of individual predictions and returns the weighted 
-    output. Later predictions are valued higher."""
-    predictions: dict = Prediction.empty_prediction_dict()
+def weighted_prediction(prediction_list: list[tuple[Prediction, ImageBox or None]]) -> tuple[Prediction, ImageBox or None]:
+    """
+    Takes the list of individual predictions and returns the weighted 
+    output. Later predictions are valued higher.
```

Comment:
Maybe we should add some explanation off why they are more important, i.e. that they are at later stages of the game motion and less likely to be incorrect.